test/native/__mocks__/styleMock.js

```diff
@@ -135,4 +135,7 @@ module.exports = {
 	buttonNoBg: {
 		color: 'orange',
 	},
+	isSelected: {
+		color: 'blue',
+	},
```

Comment:
Mocking this style is required due to rendering `BlockAlignmentUI` in the `block-alignment-control` test, right?

Reply:
Correct. Further down the tree the following code is encountered, which requires a mock of the style attribute referenced directly. https://github.com/WordPress/gutenberg/blob/e5cb098f718d7d1ee1fc2946374c9388e1f0cb2e/packages/components/src/mobile/bottom-sheet/cell.native.js#L412

Reply:
Thanks for clarifying this ❤️ !